BildCreatorLibrary/Creator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using BildingsLibrary;
+using System.Collections;
+
+namespace BildCreatorLibrary
+{
+
+    //Фабрика объектов
+    public class Creator
```

Comment:
```suggestion
    public static class Creator
```
Если все методы в классе статические, то и класс надо делать статическим.